portfolio/src/main/webapp/style.css

```diff
@@ -35,22 +34,9 @@ body {
 
 img:hover {
   opacity: 1.2;
-  animation: shake 0.5s;
-  animation-iteration-count: infinite;
-}
-
-@keyframes shake {
-  0% { transform: translate(1px, 1px) rotate(0deg); }
-  10% { transform: translate(-1px, -2px) rotate(-1deg); }
-  20% { transform: translate(-3px, 0px) rotate(1deg); }
-  30% { transform: translate(3px, 2px) rotate(0deg); }
-  40% { transform: translate(1px, -1px) rotate(1deg); }
-  50% { transform: translate(-1px, 2px) rotate(-1deg); }
-  60% { transform: translate(-3px, 1px) rotate(0deg); }
-  70% { transform: translate(3px, 1px) rotate(-1deg); }
-  80% { transform: translate(-1px, -1px) rotate(1deg); }
-  90% { transform: translate(1px, 2px) rotate(0deg); }
-  100% { transform: translate(1px, -2px) rotate(-1deg); }
+  height: 300px;
+  width: 250px;
+  border: outset;
```

Comment:
Changing the size of the image doesn't look that nice because it moves other elements further down. This would look better if the image would use up the same space before and after hovering over it. Maybe you could add margins to the image and when you hover over the image you remove the margins but make the image bigger instead.

Reply:
That's a nice idea! Will do that. Thanks.